render.py

```diff
@@ -0,0 +1,127 @@
+    indicative = root.findall('.//ns:indicative', ns)[1]
+    result += ("X" * len(str(indicative[1][0][0].text)) + ", " + "X" * len(str(indicative[1][1][0].text)) + " " + "X" * len(str(indicative[1][1][1].text)) + ". " + "0" * len(str(indicative[1][1][2].text))).ljust(66) + "".ljust(11) + "\n"
+    result += "<FORMER ADDRESS>" + "".ljust(61) + "\n"
+    former_address = indicative[2]
+    if former_address.find('.//ns:address', ns) is None:
+        result += "\n"
+    else:
+        result += ("X" * len(str(indicative[2][0][1][0].text)) + ", " + "X" * len(str(indicative[2][0][2][0].text)) + " " + "X" * len(str(indicative[2][0][2][1].text)) + ". " + "0" * len(str(indicative[2][0][2][2].text))).ljust(66) + "".ljust(11) + "\n"
+    result += "                                        <POSITION>" + "".ljust(27) + "\n"
+    result += "<CURRENT EMPLOYER AND ADDRESS                           <RPTD><HIRE>" + "".ljust(9) + "\n"
+    result += "XXXX".ljust(40) + "XX".ljust(16) + "0/00".ljust(6) + "0/00".ljust(6) + "".ljust(9) + "\n"
+    result += "<FORMER EMPLOYER AND ADDRESS>" + "".ljust(48) + "\n"
+    result += "XXXX".ljust(40) + "XX".ljust(16) + "0/00".ljust(6) + "0/00".ljust(6) + "".ljust(9) + "\n"
+    result += "\n"
+    determinationStatus = root.find('.//ns:determinationStatus', ns)
+    result += "                  ***WARNING INTERNAL USE ONLY - NOT FOR RESALE***" + "".ljust(11) + "\n"
+    result += "-" * 77 + "\n"
+    result += "S P E C I A L M E S S A G E S" + "\n"
+    flag = False
+    for alert in determinationStatus.findall('ns:redFlag', ns):
+        if str(alert.text) == 'None':
+            break
+        if flag == False:
+            result += "***Alerts***"  + "\n"
+            flag = True
+        if len(str(alert.text)) > 77:
+            result += "***" + str(alert.text)[0:74]  + "\n"
+            result += str(alert.text)[74:] + "***"  + "\n"
+        else:
+            result += "***" + str(alert.text) + "***"  + "\n"
+    flag = False
+    for warning in determinationStatus.findall('ns:warnings', ns):
+        if str(warning.text) == 'None':
+            break
+        if flag == False:
+            result += "***Warnings***"  + "\n"
+            flag = True
+        if len(str(warning.text)) > 77:
+            result += "***" + str(warning.text)[0:74] + "\n"
+            result += str(warning.text)[74:] + "***" + "\n"
+        else:
+            result += "***" + str(warning.text) + "***" + "\n"
+    result += "-" * 77 + "\n"
+    result += "M O D E L  P R O F I L E          * * * A L E R T * * *" + "\n"
+    result += "***RECOVERY MODEL 1.0 SCORE +675 : ***" + "\n"
+    result += "***NEW ACCT MODEL 2.0 ALERT: SCORE +731 : 010, 017, 031, 058 *** IN" + "\n"
+    result += "***ADDITION TO THE FACTORS LISTED ABOVE, THE NUMBER OF INQUIRIES ON THE" + "\n"
+    result += "***CONSUMER'S CREDIT FILE HAS ADVERSELY AFFECTED THE CREDIT SCORE." + "\n"
+    result += "-" * 77 + "\n"
+    result += "T R A D E S" + "\n"
```

Comment:
The public records and collections sections are missing
![image](https://user-images.githubusercontent.com/3218054/28983588-5f745dda-790f-11e7-9565-763bb3e4d65f.png)